README.md

```diff
@@ -28,8 +28,9 @@ jobs:
         args: |
           --test \
           --all \
-          --target /data \
-          --docker-hub userspace-name
+          --target addon-folder \
+          --docker-hub user-name \
+          --image {arch}-your-addon-name
```

Comment:
I'm confused? The image is read from the add-on configuration, right?

<https://github.com/home-assistant/builder/blob/master/builder.sh#L477>

Reply:
Certainly you are right.
I will change this and additionally `user-name` to `${{ secrets.DOCKERHUB_USERNAME }}`